js/Genoverse.js

```diff
@@ -109,10 +114,14 @@ var Genoverse = Base.extend({
       var js       = Genoverse.origin + 'js/plugins/' + plugin + '.js';
       var deferred = $.Deferred();
 
+      function done() {
+        browser.loadedPlugins[plugin] = browser.loadedPlugins[plugin] || 'script';
+        deferred.resolve(arg);
+      }
+
       function getCSS() {
-        function done() {
-          browser.loadedPlugins[plugin] = browser.loadedPlugins[plugin] || 'script';
-          deferred.resolve(arg);
+        if(runningInModule || true) {
+          return done();
```

Comment:
You need to check 
On line 137
`$.getScript(js, getCSS);`
As it tries to pull scripts when in storybook